Create new miniforge Blogpost (#80)
BiAPoL/blog: docs/intro.md

## Recent blog posts

### [Getting started with Miniforge and Python](mara_lampert/getting_started_with_miniforge_and_python/readme)]

<small>[Mara Lampert](mara_lampert/readme), July 8th, 2024</small><br>

> <img src="images/package_manager.jpeg" width="100" align="right"><p>This post will help you to get started with Python using Miniforge. More precisely, you will learn how to install Miniforge, how to create and use conda environments and know about some very important packages. </p>

Reviewer: ```suggestion
<img src="images/package_manager.jpeg" width="100" align="right"><p>This post will help you to get started with Python using Miniforge. More precisely, you will learn how to install Miniforge, how to create and use conda environments and you will get to know some important packages for bio image analysis.</p>
```
